lms/djangoapps/certificates/models.py

```diff
@@ -563,6 +563,17 @@ def certificate_info_for_user(user, course_id, grade, user_is_whitelisted=None):
     certificate_type = 'N/A'
     eligible_for_certificate = 'Y' if (user_is_whitelisted or grade is not None) and user.profile.allow_certificate \
         else 'N'
+    if eligible_for_certificate == 'Y':
+        # username, course_id, user_is_whitelisted, grade, allow_certificate
+        LOGGER.info("Student is marked eligible_for_certificate in grade report "
+                    "(user=%s, course_id=%s, user_is_whitelisted=%s, grade=%s, allow_certificate=%s)",
+                    user,
+                    course_id,
+                    user_is_whitelisted,
+                    grade,
```

Comment:
I think it makes more sense to log this where it's called in task_helper.py. That way, we can also tell:
* What the grade percentage was
* What the minimum passing grade cutoff is
in addition to the things you're logging here